ledger-core/virtual/integration/method_test.go

```diff
@@ -1529,3 +1527,190 @@ func TestVirtual_Method_Have_ObjectState(t *testing.T) {
 		})
 	}
 }
+
+// twice ( A.Foo -> B.Bar, B.Bar )
+func TestVirtual_CallContractTwoTimes(t *testing.T) {
+	t.Log("C5183")
+
+	mc := minimock.NewController(t)
+
+	server, ctx := utils.NewUninitializedServer(nil, t)
+	defer server.Stop()
+
+	executeDone := server.Journal.WaitStopOf(&execute.SMExecute{}, 2)
+
+	runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
+		return execution.Request.CallOutgoing.String()
+	})
+	server.ReplaceRunner(runnerMock)
+	server.Init(ctx)
+	server.IncrementPulseAndWaitIdle(ctx)
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+
+	var (
+		flags     = contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
+		callFlags = payload.BuildCallFlags(flags.Interference, flags.State)
+
+		objectAGlobal = reference.NewSelf(server.RandomLocalWithPulse())
+
+		classB        = gen.UniqueGlobalRef()
+		objectBGlobal = reference.NewSelf(server.RandomLocalWithPulse())
+
+		outgoingFirstCall  = server.RandomLocalWithPulse()
+		outgoingSecondCall = server.RandomLocalWithPulse()
+
+		outgoingReasonFirst  = reference.NewSelf(outgoingFirstCall)
+		outgoingReasonSecond = reference.NewSelf(outgoingSecondCall)
+	)
+
+	// create objects
+	{
+		Method_PrepareObject(ctx, server, payload.Ready, objectAGlobal)
+		Method_PrepareObject(ctx, server, payload.Ready, objectBGlobal)
+	}
+
+	// add ExecutionMocks to runnerMock
+	{
+		firstBuilder := execution.NewRPCBuilder(outgoingReasonFirst, objectAGlobal)
+		objectAExecutionFirstMock := runnerMock.AddExecutionMock(outgoingFirstCall.String())
+		objectAExecutionFirstMock.AddStart(
+			func(ctx execution.Context) {
+				t.Log("ExecutionStart [A.Foo] first call")
```

Comment:
Do not use direct call t.Log. Use our logger as descried [here](https://insolar.atlassian.net/wiki/spaces/DEV/pages/1431568423/Core+v2+-+Logger+in+tests)

Reply:
removed